docs/open-source/create-a-provider.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Create a provider
+description: A short guide on integrating Qiskit into an external provider's quantum resources.     
+---
+# Integrate external quantum resources with Qiskit
+
+The Qiskit SDK is built to support third parties in creating external providers of quantum resources. 
+
+This means that any organization which develops or deploys quantum compute resources can integrate their services into Qiskit and tap into its userbase. 
+
+Doing so requires creating a package which supports requests for quantum compute resources and returns them to the user. 
+
+Additionally, the package must allow users to submit jobs and retrieve their results through an implementation of the `qiskit.primitives` objects.
+
+## Providing access to backends
+In order for users to transpile and execute `QuantumCircuit` objects using external resources, they need to instantiate an object containing a [`Target`](../api/qiskit/qiskit.transpiler.Target) which provides information about a QPU's constraints such as its connectivity, basis gates, and number of qubits. This can be provided through an interface similar to the [`QiskitRuntimeService`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService) through which a user can make requests for a QPU. This object should, at minimum, contain a `Target`, but a simpler approach would be to return an object which has inherited from either the [`BackendV2`](../api/qiskit/qiskit.providers.BackendV2) or [`BackendV1`](../api/qiskit/qiskit.providers.BackendV1) classes.
```

Comment:
BackendV1 is deprecated and its usage is not recommended at this point https://github.com/Qiskit/qiskit/pull/12629/

```suggestion
In order for users to transpile and execute `QuantumCircuit` objects using external resources, they need to instantiate an object containing a [`Target`](../api/qiskit/qiskit.transpiler.Target) which provides information about a QPU's constraints such as its connectivity, basis gates, and number of qubits. This can be provided through an interface similar to the [`QiskitRuntimeService`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService) through which a user can make requests for a QPU. This object should, at minimum, contain a `Target`, but a simpler approach would be to return a [`BackendV2`](../api/qiskit/qiskit.providers.BackendV2) instance.
```